### PR TITLE
fix: TR-855 fix bottom toolbar behavior on shrink display

### PR DESCRIPTION
**Pull request by lecosson**
Related to:
- https://oat-sa.atlassian.net/browse/TR-855

Depencies: 
- https://github.com/oat-sa/tao-test-runner-qti-fe/pull/367

Fix disappers part of bottom toolbar on shrink display.

### CODE REVIEW

File: views/package.json [hunk 1]
```diff
@@ -9,6 +9,6 @@
   "license": "GPL-2.0",
   "dependencies": {
     "@oat-sa/tao-test-runner": "0.8.1",
-    "@oat-sa/tao-test-runner-qti": "2.20.0"
+    "@oat-sa/tao-test-runner-qti": "github:oat-sa/tao-test-runner-qti#fix/TR-855/control-box-elements-visibility"
```
Comment (btamas) [inline views/package.json:12]: Wrong repo name. Why do you do it manually? Just install like `npm i github:oat-sa/tao-test-runner-qti-fe#fix/TR-855/control-box-elements-visibility`

**Comment by btamas** [comment]
js bundle should be ran as well, to be able to test without debug mode, for example in IE11. BTW, it works fine.

**Comment by btamas** [comment]
**Review Checklist**
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful
- [ ] Pull request's target is not `master`
- [ ] Changelog is updated according to changes (if applicable)
- [ ] Documentation is updated according to changes (if applicable)

**Review by btamas — APPROVED** [review]

**Review by meefox — APPROVED** [review]
Review is done in main PR

**Pull request merged**